sqlx-core/Cargo.toml

```diff
@@ -22,7 +22,9 @@ json = ["serde", "serde_json"]
 _rt-async-std = ["async-std", "async-io"]
 _rt-tokio = ["tokio", "tokio-stream"]
 _tls-native-tls = ["native-tls"]
-_tls-rustls = ["rustls", "rustls-pemfile", "webpki-roots"]
+_tls-rustls-aws-lc-rs = ["__tls-rustls", "rustls/aws-lc-rs"]
+_tls-rustls-ring = ["__tls-rustls", "rustls/ring"]
+__tls-rustls = ["rustls", "rustls-pemfile", "webpki-roots"]
```

Comment:
The extra underscore here is weird. Intentional?

Reply:
It is intentional, to signify that this is sort of crate private, not just workspace-private. Happy to take suggestions for a different name!

Reply:
The problem is, it just looks like a typo. I'd just go with the single underscore like everything else. 

We don't need a distinction between crate-private and workspace-private features.